test/controllers/api/v2/repositories_controller_test.rb

```diff
@@ -50,13 +50,15 @@ def assert_response_ids(response, expected)
     end
 
     def test_index_with_product_id
-      ids = Repository.where(:product_id => @product.id, :library_instance_id => nil).pluck(:id)
+      ids = Repository.where(:product_id => @product.id, :library_instance_id => nil).where(:content_view_version_id => @organization.default_content_view.versions.first.id).pluck(:id)
 
       response = get :index, :product_id => @product.id, :organization_id => @organization.id
+      response_ids = JSON.parse(response.body)['results'].map { |repo| repo['id'] }
 
+      debugger
```

Comment:
I was more trying to point out the wonderful soul who contributed the debugger cop to rubocop.